bbb-libreoffice/assets/convert-cool.sh

```diff
@@ -16,7 +16,7 @@ HOST=127.0.0.1
 
 # Set this to "-k" to allow it to work in a test environment, ie with a self signed
 # certificate
-UNSECURE=
+INSECURE=
```

Comment:
```suggestion
UNSECURE=
```

Let's keep one as-is as it's both a property (and renaming it can break tools) and also somewhat correct - I've seen unsecure for applications